<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="2" passed="0" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-03-25T13:14:58 IST" name="Default suite" finished-at="2024-03-25T13:15:34 IST" duration-ms="35925">
    <groups>
      <group name="RegressionSuite">
        <method signature="SearchContactTest.searchContact1()[pri:0, instance:contactsTests.SearchContactTest@77888435]" name="searchContact1" class="contactsTests.SearchContactTest"/>
        <method signature="SearchContactTest.searchContact2()[pri:0, instance:contactsTests.SearchContactTest@77888435]" name="searchContact2" class="contactsTests.SearchContactTest"/>
      </group> <!-- RegressionSuite -->
      <group name="SmokeSuite">
        <method signature="SearchContactTest.searchContact1()[pri:0, instance:contactsTests.SearchContactTest@77888435]" name="searchContact1" class="contactsTests.SearchContactTest"/>
        <method signature="SearchContactTest.searchContact2()[pri:0, instance:contactsTests.SearchContactTest@77888435]" name="searchContact2" class="contactsTests.SearchContactTest"/>
      </group> <!-- SmokeSuite -->
    </groups>
    <test started-at="2024-03-25T13:14:58 IST" name="Default test" finished-at="2024-03-25T13:15:34 IST" duration-ms="35925">
      <class name="contactsTests.SearchContactTest">
        <test-method is-config="true" signature="bsConfig()[pri:0, instance:contactsTests.SearchContactTest@77888435]" started-at="2024-03-25T13:14:58 IST" name="bsConfig" finished-at="2024-03-25T13:14:58 IST" duration-ms="9" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bsConfig -->
        <test-method is-config="true" signature="bcConfig()[pri:0, instance:contactsTests.SearchContactTest@77888435]" started-at="2024-03-25T13:14:59 IST" name="bcConfig" finished-at="2024-03-25T13:15:06 IST" duration-ms="7615" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bcConfig -->
        <test-method is-config="true" signature="bmConfig()[pri:0, instance:contactsTests.SearchContactTest@77888435]" started-at="2024-03-25T13:15:07 IST" name="bmConfig" finished-at="2024-03-25T13:15:09 IST" duration-ms="2961" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bmConfig -->
        <test-method signature="searchContact1()[pri:0, instance:contactsTests.SearchContactTest@77888435]" started-at="2024-03-25T13:15:09 IST" name="searchContact1" finished-at="2024-03-25T13:15:16 IST" duration-ms="6820" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[No contact found with lastname Chaithra
 expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: No contact found with lastname Chaithra
 expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at contactsTests.SearchContactTest.searchContact1(SearchContactTest.java:29)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- searchContact1 -->
        <test-method is-config="true" signature="amConfig()[pri:0, instance:contactsTests.SearchContactTest@77888435]" started-at="2024-03-25T13:15:17 IST" name="amConfig" finished-at="2024-03-25T13:15:19 IST" duration-ms="2775" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- amConfig -->
        <test-method is-config="true" signature="bmConfig()[pri:0, instance:contactsTests.SearchContactTest@77888435]" started-at="2024-03-25T13:15:19 IST" name="bmConfig" finished-at="2024-03-25T13:15:21 IST" duration-ms="2085" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- bmConfig -->
        <test-method signature="searchContact2()[pri:0, instance:contactsTests.SearchContactTest@77888435]" started-at="2024-03-25T13:15:21 IST" name="searchContact2" finished-at="2024-03-25T13:15:28 IST" duration-ms="6950" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[No contact found with lastname Neeraja
 expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: No contact found with lastname Neeraja
 expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at contactsTests.SearchContactTest.searchContact2(SearchContactTest.java:48)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- searchContact2 -->
        <test-method is-config="true" signature="amConfig()[pri:0, instance:contactsTests.SearchContactTest@77888435]" started-at="2024-03-25T13:15:29 IST" name="amConfig" finished-at="2024-03-25T13:15:34 IST" duration-ms="5035" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- amConfig -->
        <test-method is-config="true" signature="acConfig()[pri:0, instance:contactsTests.SearchContactTest@77888435]" started-at="2024-03-25T13:15:34 IST" name="acConfig" finished-at="2024-03-25T13:15:34 IST" duration-ms="570" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- acConfig -->
        <test-method is-config="true" signature="afterSuite()[pri:0, instance:contactsTests.SearchContactTest@77888435]" started-at="2024-03-25T13:15:35 IST" name="afterSuite" finished-at="2024-03-25T13:15:35 IST" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterSuite -->
      </class> <!-- contactsTests.SearchContactTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
